### ISSUE TITLE
[enhancement] custom sort order

### ISSUE THREAD

**Issue by kyle-ilantzis** [nicholasrout/lens-launcher#11]
the sort order "most used" is sometimes confusing because apps will move around.

A custom sort order would let me place my favorite apps near the top of the screen so i can easily find them. 

Also this will leave apps at the same place, they wont move around and confuse me.

the tap "apps" could have a reorder icon on each list item to allow this

**Comment by CreaRo** [comment]
Yup. This is something we're looking to implement in the near future. Thanks!

**Comment by nicholasrout** [comment]
Hi @kyle-ilantzis, thanks for the suggestion. Will work on getting this in to a future release

**Comment by nicholasrout** [comment]
I plan to implement this via long-press drag to rearrange
